src/fragments/lib/geo/js/geofences.mdx

```diff
@@ -101,6 +101,37 @@ And your app CSS should be updated to show the sign out button
 
 ![Geofence Edit](/images/geofence-edit.png)
 
+### NextJS and Polyfills
+
+<Callout>
+
+**Note:** The current version of `maplibre-gl-draw-circle` that we depend on uses a slightly older version of `mapbox-gl-draw`. `mapbox-gl-draw` v1.1.1 will require some polyfills in NextJS to get your application to compile otherwise you will get an error like:
+
+`Uncaught ReferenceError: process is not defined`
```

Comment:
should this be a multiline code block ?

Reply:
I posted a screenshot above, not sure if it's necessary to be a multiline block